### PR TITLE
(feat) Add radiology review dialog

### PR DESCRIPTION
**Pull request by gabriel090**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.
Review the Radiology Report prompt.
## Summary

Screenshot before modification on the Radiology report
RadiologyReportBefore

Screenshot after modification was done
RadiologyReportAfter

### CODE REVIEW

**Comment by ojwanganto** [comment]
Thanks, @gabriel090. Please check the failing yarn checks. They are easy ones.

**Comment by jecihjoy** [comment]
@gabriel090 one last thing then we merge this, on procedure report, we should not be seeing lab details,
also the report should be coming from the `order?.procedures[0]?.procedureReport` field

**Comment by ojwanganto** [comment]
@gabriel090 could you please address all the lint issues in this PR?

File: src/radiology-tabs/test-ordered/pick-radiology-order/add-to-worklist-dialog.resource.ts [hunk 1]
```diff
@@ -135,3 +135,16 @@ export async function updateOrder(uuid: string, body: any) {
     body: body,
   });
 }
+
+// update Procedure
+export async function updateProdedure(uuid: string, body: any) {
```
Comment (donaldkibet) [inline src/radiology-tabs/test-ordered/pick-radiology-order/add-to-worklist-dialog.resource.ts:140]: we should avoid using any

File: src/radiology-tabs/work-list/work-list.resource.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { openmrsFetch } from "@openmrs/esm-framework";
 import useSWR from "swr";
 export interface Result {
+  [x: string]: any;
```
Comment (donaldkibet) [inline src/radiology-tabs/work-list/work-list.resource.ts:4]: Please use descriptive names for `x`

File: src/radiology-tabs/work-list/work-list.resource.ts [hunk 3]
```diff
@@ -34,8 +35,14 @@ export interface Result {
   links: Link[];
   type: string;
   resourceVersion: string;
+  procedures: Procedure[];
```
Comment (donaldkibet) [inline src/radiology-tabs/work-list/work-list.resource.ts:38]:
```suggestion
  procedures: Array<Procedure>;
```

File: src/utils/functions.ts [hunk 4]
```diff
@@ -244,3 +244,5 @@ export function useGetPatientByUuid(uuid: string) {
     isError: error,
   };
 }
+
+export const BLEEDING_SITE = "162668AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
```
Comment (donaldkibet) [inline src/utils/functions.ts:248]: Lets use config here, we would not want to change code later if this concept changes

File: src/radiology-tabs/review-ordered/review-radiology-report-dialog/review-radiology-report-dialog.component.tsx [hunk 5]
```diff
@@ -78,10 +95,40 @@ const ReviewOrderDialog: React.FC<ReviewOrderDialogProps> = ({
                 <AccordionItem
                   title={t("procedureInstructions", "Procedure Instructions")}
                 >
-                  <p>Lorem ipsum dolor sit amet</p>
+                  <p>
```
Comment (donaldkibet) [inline src/radiology-tabs/review-ordered/review-radiology-report-dialog/review-radiology-report-dialog.component.tsx:98]: We should not be wrap this in `<p>`

File: src/radiology-tabs/review-ordered/review-radiology-report-dialog/review-radiology-report-dialog.component.tsx [hunk 6]
```diff
@@ -28,45 +32,58 @@ const ReviewOrderDialog: React.FC<ReviewOrderDialogProps> = ({
   const { t } = useTranslation();
 
   const [notes, setNotes] = useState("");
```
Comment (donaldkibet) [inline src/radiology-tabs/review-ordered/review-radiology-report-dialog/review-radiology-report-dialog.component.tsx:34]: This state is not being used, please remove it

**Review by donaldkibet — APPROVED** [review]
LGTM @gabriel090 please address post merge comments above

File: src/radiology-tabs/work-list/work-list.resource.ts (hunk 3 above)
Comment (donaldkibet) [reply src/radiology-tabs/work-list/work-list.resource.ts:38]: The convention in OpenMRS is to use `Array<>` for this type of declarations of types

**Pull request merged**